packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -345,6 +345,7 @@ class HtmlElementView extends StatelessWidget {
     super.key,
     required this.viewType,
     this.onPlatformViewCreated,
+    this.creationParams,
```

Comment:
This new parameter needs a little bit of documentation on the docblock, because it's published [here](https://api.flutter.dev/flutter/widgets/HtmlElementView-class.html).

(Also, does the docblock represent how the widget works? the contents are a little bit weird)

Reply:
> This new parameter needs a little bit of documentation on the docblock, because it's published [here](https://api.flutter.dev/flutter/widgets/HtmlElementView-class.html).

Do you have some ideas of what we can add to the docs?

> (Also, does the docblock represent how the widget works? the contents are a little bit weird)

Not sure what you mean. What I'm trying to do in the docs is explain the connection between this and the view factory since that's the important part for users to understand. Everything else that happens inside this widget (e.g. the controller) is an implementation detail.

Reply:
Never mind, my rant is more about the documentation of the `class HtmlElementView`, your documentation for the params object seems OK to me.